llvm/lib/ObjCopy/ELF/ELFObjcopy.cpp

```diff
@@ -292,6 +292,8 @@ static Error updateAndRemoveSymbols(const CommonConfig &Config,
     return Error::success();
 
   Obj.SymbolTable->updateSymbols([&](Symbol &Sym) {
+    if (ELFConfig.SymbolsToSkip.matches(Sym.Name))
+      return;
```

Comment:
Nit: let's add a blank line after this line to break it away from the following comment.